gnosis/safe/safe.py

```diff
@@ -88,9 +88,12 @@ class Safe:
         0x4A204F620C8C5CCDCA3FD54D003BADD85BA500436A431F0CBDA4F558C93C34C8
     )
     # keccak256("EIP712Domain(uint256 chainId,address verifyingContract)");
-    DOMAIN_TYPEHASH = bytes.fromhex(
+    DOMAIN_TYPEHASH_V1_3_0 = bytes.fromhex(
         "47e79534a245952e8b16893a336b85a3d9ea9fa8c573f3d803afb92a79469218"
     )
+    DOMAIN_TYPEHASH_V1_1_1 = bytes.fromhex(
+        "035aff83d86937d35b32e04f0ddc6ff469290eef2f1b692d8a815c89404d4749"
```

Comment:
I'd add a similar comment on what resulted in this hash value:

```python
# keccak256("EIP712Domain(address verifyingContract)")
```

Reply:
Done https://github.com/safe-global/safe-eth-py/pull/422/commits/f1ac04392efe6c31df0dc636a931ade86c92155e